Server/ChessArena.proto

```diff
@@ -0,0 +1,68 @@
+syntax = "proto3";
+
+service ChessArena {
+    // A simple RPC
+    //
+    // Inform server that the client is ready to start the game. 
+    // Returns one of the following:
+    // 1. Player is starting player (white). Opponent move message not set.
+    // 2. Player is not starting player (black). Opponent move message provides opening
+    //    move coordinates. 
+    rpc Initialize(PlayerInformation) returns (GameStartInformation) {}
+
+    // A Bidirectional streaming RPC.
+    //
+    // Send player moves. Receive opponent moves.
+    rpc CreateMovements(stream PlayerMove) returns (stream OpponentMove) {}
```

Comment:
Is there benefits on using bi-directional streaming RPC?

Reply:
It sounded most fitting for the continuous messaging. gRPC core concepts states following for bidirectional:
_For example, a server can wait until it has received all of a client’s messages before writing its messages, or the server and client can play “ping-pong” – the server gets a request, then sends back a response, then the client sends another request based on the response, and so on._

Do you think there is some downsides?

Reply:
This leaves it for the players to handle the orchestration of writes/reads to/from the stream. But in this case this should be pretty straight forward, so I think we can start experimenting with it and learn on it's usage. We can always roll back to non-streaming calls if we see trouble.